.github/actions/create-package/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+PACKAGE_CLOUD_API_KEY="$1"
+
+# Run the build command
+case "$ARCHITECTURE" in
```

Comment:
The name architecture is a little bit confusing, maybe try `DOMAIN_SIZE`?

Reply:
Changed `ARCHITECTURE` to `DOMAIN_SIZE`.